docs/recipes/vue.md

```diff
@@ -32,8 +35,16 @@ The first step is setting up a helper to configure the environment to transpile
 ```js
 // ./test/_setup.js
 
+// https://github.com/vuejs/vue-test-utils/issues/936
+// from @vue/cli-plugin-unit-mocha/setup.js
+require('jsdom-global')()
+
+// https://github.com/vuejs/vue-test-utils/issues/936
+// better fix for "TypeError: Super expression must either be null or
+// a function" than pinning an old version of prettier.
+window.Date = Date
```

Comment:
The issue you're linking to seems specific to rollup? How does that impact testing? Why is the comment referring to prettier?

Not saying that a workaround is wrong but between `browser-env`, `jsdom-global` and this it doesn't make it clear what setup is actually necessary, nor how we'd know that this workaround is no longer needed.

Reply:
this link is more about the tool `vue-test-utils` and the problem that arose in the test runner `Mocha` , I encountered the same problem when using `ava`. I did not go into details about the origin of this error, but if I need it, I will do it but it will take me time. At this stage, I wanted to save people from finding solutions when using `ava` associated with this error